.github/workflows/lighthouse.yml

```diff
@@ -1,25 +1,40 @@
 name: DCR Run Lighthouse CI
 on:
-    push:
-        paths-ignore:
-            - "apps-rendering/**"
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - "apps-rendering/**"
+  pull_request:
+    types: [synchronize]
+
 jobs:
-    lhci:
-        name: DCR Lighthouse
-        runs-on: ubuntu-latest
-        steps:
-            - name: Checkout code
-              uses: actions/checkout@v3
-            - name: Install Node
-              uses: guardian/actions-setup-node@main
-            # Make sure we install dependencies in the root directory
-            - uses: bahmutov/npm-install@v1
-            - run: make build
-              working-directory: dotcom-rendering
-            - name: Install and run Lighthouse CI
-              working-directory: dotcom-rendering
-              env:
-                  LHCI_GITHUB_TOKEN: ${{ secrets.LHCI_GITHUB_TOKEN }}
-              run: |
-                  npm install -g puppeteer-core@2.1.0 @lhci/cli@0.8.2
-                  lhci autorun
+  lhci:
+    name: DCR Lighthouse
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Install Node
+        uses: guardian/actions-setup-node@main
+      # Make sure we install dependencies in the root directory
+      - uses: bahmutov/npm-install@v1
+      - run: make build
+        working-directory: dotcom-rendering
+      - name: Install and run Lighthouse CI
+        working-directory: dotcom-rendering
+        env:
+          LHCI_GITHUB_TOKEN: ${{ secrets.LHCI_GITHUB_TOKEN }}
+        run: |
+          npm install -g puppeteer-core@2.1.0 @lhci/cli@0.8.2
+          lhci autorun --assert.includePassedAssertions
```

Comment:
ah, so before it was only rendering results for the failed budgets? that makes sense!

fwiw it looks as though we can include this flag in the [lighthouserc.js](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/lighthouserc.js) config instead, if we want to (see https://github.com/GoogleChrome/lighthouse-ci/blob/main/docs/configuration.md#includepassedassertions). I don't think it makes a huge difference either way but might be good to keep all of the config in one place?